app/permissions_migrations.go

```diff
@@ -152,7 +152,7 @@ func (a *App) doPermissionsMigration(key string, migrationMap permissionsMap) *m
 	return nil
 }
 
-func getEmojisPermissionsSplitMigration() permissionsMap {
+func getEmojisPermissionsSplitMigration(a *App) (permissionsMap, error) {
```

Comment:
should these be functions _on_ the `App` as opposed to functions that take `App` as a parameter? seems like passing `App` as a parameter is not something we ever do - im not sure what the benefit / downsides are of doing it either way in this case are though... wondering if maybe you did it for a specific reason